Makefile

```diff
@@ -13,7 +13,8 @@ S3_RELEASE_BUCKET = $$S3_RELEASE_BUCKET
 BUILDNUMBER      ?= $(shell ./scripts/compute_build_number.sh)
 COMMITHASH       := $(shell ./scripts/compute_build_commit.sh)
 BUILDBRANCH      := $(shell ./scripts/compute_branch.sh)
-BUILDCHANNEL     := $(shell ./scripts/compute_branch_channel.sh $(BUILDBRANCH))
+CHANNEL          ?= $(shell ./scripts/compute_branch_channel.sh $(BUILDBRANCH))
+BUILDCHANNEL     :=  CHANNEL
```

Comment:
maybe I'm missing something, but what is the importance of having the BUILDCHANNEL at all at that point ?
it looks like you could have just used the CHANNEL variable all along, while declaring it as 
```CHANNEL ?= $(shell ./scripts/compute_branch_channel.sh $(BUILDBRANCH))```

Reply:
I considered removing BUILDCHANNEL, however, it is referenced in the scripts.  Cleaning that up is out of the scope of this PR which is focused on fixing the perf tests.

Reply:
so, can we use just BUILDCHANNEL instead ? having both BUILDCHANNEL and CHANNEL is confusing.

Reply:
CHANNEL is more widely used.

Reply:
My preference is to just do the change now.  By introducing this here, it is bringing it into scope, and this kind of variable assignment is really confusing (especially when there's no context given in the form of an comment).

I just grepped for `BUILDCHANNEL` and it doesn't look at all like it would introduce any risk by doing a simple search and replace.

Reply:
Replaced all references to $BUILDCHANNEL with $CHANNEL.

Reply:
Great, thanks